### ISSUE TITLE
API additions

### ISSUE THREAD

**Issue by BrendanBenshoof** [UrDHT/PyUrDHT#5]
After dealing with real world problems.
I think we should add:

~~\- [ ] ipv6 support <- this leads to partitioning issues! do all nodes need ipv6? proxying via shared protocol nodes?~~
- [x] "what is my ip?" request on the REST api to help nodes bootstrap
- [ ] "what is your info" request on the REST api

**Comment by nwautomator** [comment]
Node public IP info is now available as per issue #10. Note that it only provides the node's IP. We'll need to add the desired logic if we need anything else.

**Comment by mmcd** [comment]
[this](http://haxpo.nl/materials/haxpo2015ams/D3%20-%20R.%20Schaefer%20and%20J.%20Salazar%20-%20Pentesting%20in%20the%20Age%20of%20IPv6.pdf) just in on hackernews on ipv6

TLDR: ipv6 is still under dev and will change, its complicated, know your stuff before you impmlement it

**Comment by BrendanBenshoof** [comment]
ipv6 support deemed not needed/advised for now.
